### ISSUE TITLE
Issues with uninstallation

### ISSUE THREAD

**Issue by MiriamJo** [nitroshare/nitroshare-desktop#271]
I am trying to remove the installed nitroshare completely from my windows 11 computer. However, it sais theres a file still open which i need to close first. I don't know which files are there to close, since the task manager is completely "empty".

The automatic windows uninstallation process just told me "There are still files that can be uninstalled manually". Please help me, since I cant use it anymore in this state. I just want to reinstall it.

**Comment by MiriamJo** [comment]
It told me "Network File Transfer Application" is still using it. I am not sure on how to kill it.

**Comment by nathan-osman** [comment]
It sounds like NitroShare is still running - if you check the system tray, do you see an icon for it?

**Comment by MiriamJo** [comment]
Yes I thought so, but I made sure all connections were closed before trying to uninstall / delete the nitroshare folder. So I manually closed the Programm with right click ->"close".
